about_scoring_project.rb

```diff
@@ -30,7 +30,27 @@
 # Your goal is to write the score method.
 
 def score(dice)
-  # You need to write this method
+  return 0 if dice == []
+  dice = dice.sort  
+  points_scored = 0
+
+  if dice[1] == dice[0] && dice[2] == dice[0]
```

Comment:
What would if [1,5,5,5,5]? I think you should process array without bindings to sequence.